code/game/objects/items/devices/uplinks.dm

```diff
@@ -255,7 +255,7 @@ var/list/world_uplinks = list()
 	
 //Refund proc for the borg teleporter (later I'll make a general refund proc if there is demand for it)
 /obj/item/device/radio/headset/syndicate/attackby(obj/item/weapon/W as obj, mob/user as mob, params)
```

Comment:
the more I look at this, the more the fact this has to be done by a headset seems wrong to me

the uplinks that spawn in syndicate backpacks (which have hidden uplinks in them) are "/obj/item/device/radio/uplink".